source/02-FN/05.ptx

```diff
@@ -397,10 +397,18 @@ https://activecalculus.org/prelude/sec-changing-composite.html#act-changing-comp
   <introduction>
     <p>
 Let functions <m>p</m> and <m>q</m> be given by the graphs below. </p>
-  
-<p>
-    INSERT GRAPH (OR SAGEPLOT VERSION) OF GRAPH FROM <url href="https://activecalculus.org/prelude/sec-changing-composite.html#act-changing-composite-tables-graphs"></url>
-</p>
+
+<image>
+  <sageplot>
+    p=line([(-4,5), (-1,-1)],color="blue",thickness=3,aspect_ratio=1,xmin=-3.5,xmax=3.5,ymin=-3.5,ymax=3.5)
```

Comment:
Can we add a label for each function? The blue one is p(x) and the green is q(x)?